bindata/manifests/sriov-config-service/kubernetes/sriov-config-service.yaml

```diff
@@ -0,0 +1,15 @@
+contents: |
+  [Unit]
+  Description=Configures SRIOV NIC
+  Wants=network-pre.target
+  Before=network-pre.target
+  
+  [Service]
+  Type=oneshot
+  ExecStart=/var/lib/sriov/sriov-network-config-daemon service -v {{ .LogLevel }}
+  StandardOutput=journal+console
+  
+  [Install]
+  WantedBy=multi-user.target
+enabled: true
+name: sriov-config.service
```

Comment:
Just thinking out loud, would we want to add templatisation for dependent services? Is it possible that some users may want to add services in the `After=` field based on their host OS setup?

Reply:
That is a great Idea the question is how we will expose that template to the user?

Reply:
we definately need to have this configurable.  Let's think about thi in a follow-up PR